apps/emqx_license/src/emqx_license.erl

```diff
@@ -193,7 +193,7 @@ is_new_client(ClientId) ->
     %% it's a new client if no live session is found
     [] =:= emqx_cm:lookup_channels(ClientId).
 
-is_max_clients_exceeded(MaxClients) when MaxClients =< ?DEFAULT_TRIAL_SESSIONS_LIMIT ->
+is_max_clients_exceeded(MaxClients) when MaxClients =< ?DEFAULT_MAX_SESSIONS_CTYPE3 ->
```

Comment:
Nit:  may be a comment to make logic more explicit
* why we compare limits with limits
* why we apply ctype-specific const (`DEFAULT_MAX_SESSIONS_CTYPE3`) to seemingly a common case, not related to a customer or license type.

Reply:
Thanks.
This is fixed in the next commit (which was sent to another branch to trigger a new CI check before, now pushed to this PR)